File: Atributos de validación Data Annotations multiidioma/[C#]-Atributos de validación Data Annotations multiidioma/C#/LocalizeDataAnnotations/Infrastructure/TextosLoader.cs

namespace LocalizeDataAnnotations.Infrastructure
{

	public static class TextosLoader
	{

			public static string Apellido
		{
			get { return TextosBBDD.Recuperar("Apellido"); }
		}
		public static string Ciudad
		{
			get { return TextosBBDD.Recuperar("Ciudad"); }
		}
		public static string CiudadErrorLongitud
		{
			get { return TextosBBDD.Recuperar("CiudadErrorLongitud"); }
		}
		public static string Email
		{
			get { return TextosBBDD.Recuperar("Email"); }
		}
		public static string Nombre
		{
			get { return TextosBBDD.Recuperar("Nombre"); }
		}
		public static string NombreObligatorio
		{
			get { return TextosBBDD.Recuperar("NombreObligatorio"); }
		}

	}

}

File: Atributos de validación Data Annotations multiidioma/[VB]-Atributos de validación Data Annotations multiidioma/VB/LocalizeDataAnnotations/Infrastructure/TextosLoader.vb

Public NotInheritable Class TextosLoader

	Public Shared ReadOnly Property Apellido() As String
		Get
			Return TextosBBDD.Recuperar("Apellido")
		End Get
	End Property
	Public Shared ReadOnly Property Ciudad() As String
		Get
			Return TextosBBDD.Recuperar("Ciudad")
		End Get
	End Property
	Public Shared ReadOnly Property CiudadErrorLongitud() As String
		Get
			Return TextosBBDD.Recuperar("CiudadErrorLongitud")
		End Get
	End Property
	Public Shared ReadOnly Property Email() As String
		Get
			Return TextosBBDD.Recuperar("Email")
		End Get
	End Property
	Public Shared ReadOnly Property Nombre() As String
		Get
			Return TextosBBDD.Recuperar("Nombre")
		End Get
	End Property
	Public Shared ReadOnly Property NombreObligatorio() As String
		Get
			Return TextosBBDD.Recuperar("NombreObligatorio")
		End Get
	End Property

End Class